### ISSUE TITLE
BUG - image-classification-using-transfer-learning-in-pytorch

### ISSUE THREAD

**Issue by kooscode** [spmallick/learnopencv#517]
I submitted a PR #516 to fix a bug with the sample code.

the code has a bug that will set output of last layer (prior to softmax) to one less than classes and that breaks training.
